test-case/check-playback.sh

```diff
@@ -94,6 +104,17 @@ do
                     dmesg > $LOG_ROOT/aplay_error_${dev}_$i.txt
                     dloge "aplay on PCM $dev failed at $i/$loop_cnt."
                     exit 1
+                else
+                    #sleep random_min ~ random_max seconds before running aplay, then aplay can be run under dsp active or suspended status randomly
```

Comment:
here not the randomly, it should be dsp status is 'active' or 'suspended'  
so it means the other status is not our expect